### PR TITLE
Fix issue #896 - Create a Brick for Links

### LINKED ISSUE #896: Create a Brick for Links
All users (99%)  want to be able to share links in their app. 
They want the link to be either : 
- Internal -> will link to bottom of app or up (avoiding scrolling)
- External

### PR DESCRIPTION
**Pull request by alicoding**
Closes #896

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson can you R? please

File: blocks/link/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    className: 'link',
+    template: require('./index.html'),
+    data: {
+        name: 'Link',
+        icon: 'images/blocks_link.svg',
+        attributes: {
+            link: {
+                label: 'Link Text',
```
Comment (k88hudson) [inline blocks/link/index.js]: This should be URL

File: blocks/link/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                type: 'input',
+                value: '',
+                placeholder: 'https://mozilla.org'
+            },
+            text: {
+              label: 'URL',
```
Comment (k88hudson) [inline blocks/link/index.js]: This should be Link Text

**Comment by k88hudson** [comment]
@alicoding mostly looks good just need to switch those labels

**Comment by alicoding** [comment]
@k88hudson R?

**Comment by thisandagain** [comment]
On Android / Chrome when you add a link, it seems to get obscured by neighboring elements. Most likely this is because it needs to be explicitly defined as a `block` level element.

screen shot 2015-02-10 at 8 05 51 pm

**Comment by alicoding** [comment]
I actually forgot to add padding to the block so it just happen to look empty when the user didn't have any text in there at all.

A question though @thisandagain should we allow empty string on both input at all? Should we make this as required field?

**Comment by alicoding** [comment]
screen shot 2015-02-10 at 23 28 14
screen shot 2015-02-10 at 23 28 17

**Comment by thisandagain** [comment]
R+ :baby_chick: :computer: :zap:

**Comment by thisandagain** [comment]
Thanks @alicoding !

**Pull request merged**